messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -498,16 +498,16 @@ public Builder spanFactory(@Nonnull EventProcessorSpanFactory spanFactory) {
         }
 
         /**
-         * Sets the {@link StreamableMessageSource} (e.g. the {@code EventStore}) which this {@link EventProcessor} will
+         * Sets the {@link StreamableEventSource} (e.g. the {@code EventStore}) which this {@link EventProcessor} will
          * track.
          *
-         * @param messageSource the {@link StreamableMessageSource} (e.g. the {@code EventStore}) which this
-         *                      {@link EventProcessor} will track
+         * @param eventSource the {@link StreamableEventSource} (e.g. the {@code EventStore}) which this
```

Comment:
Super nit: would you mind updating the JavaDoc style to use sentence-based `@param` and `@return` statements for the methods you touch? :angel:

Reply:
Yes, I'm trying to remember about that :)

Reply:
Thank you!!